tv_apps/browser/js/browser_db.js

```diff
@@ -55,6 +40,7 @@ var BrowserDB = {
    * Populate topsites data from mozSettings to database.
    */
   initSingleVariant: function browserDB_initSingleVariant() {
+/* TODO: mozSettings to indexedDB
```

Comment:
Seems they are preventing initializing browserDB. Will we restore it or just remove it?